common/djangoapps/third_party_auth/settings.py

```diff
@@ -61,6 +61,7 @@ def apply_settings(django_settings):
         'third_party_auth.pipeline.set_id_verification_status',
         'third_party_auth.pipeline.set_logged_in_cookies',
         'third_party_auth.pipeline.login_analytics',
+        'third_party_auth.pipeline.emit_login_event',
```

Comment:
I believe you may be able to update `SOCIAL_AUTH_PIPELINE` setting by configuration. In that case we wouldn't need to modify the code.

Reply:
done